src/app/app.component.ts

```
@@ -154,17 +167,34 @@ export class AppComponent {
         console.log("Show Strip");
         break;
       case "nativeAd":
-        const templateId : string = "batch-template2";
-        this.EMMA.inAppMessage({type:IN_APP_TYPE.NATIVE_AD, templateId});
-        console.log("Show Native Ad");
+        const templateId : string = "template2";
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)